src/main/scala/tech/sourced/gemini/Gemini.scala

```diff
@@ -122,6 +124,13 @@ class Gemini(session: SparkSession, log: Slf4jLogger, keyspace: String = Gemini.
     ReportExpandedGroup(duplicates)
   }
 
+  def findConnectedComponents(conn: Session): Unit = {
```

Comment:
AFAICS, it would be reasonable to have concrete return type of `Map[Int, Set[Int]]` as part of gemini API here.